doc/modules/classes.rst

```diff
@@ -1225,6 +1225,24 @@ Model validation
    pipeline.make_pipeline
    pipeline.make_union
 
+.. _partial_dependence_ref:
+
+:mod:`sklearn.partial_dependence`: Partial dependence plots
```

Comment:
Is it worth considering whether we want this to be in a subpackage with other things? It relates to metrics, but isn't really relevant there because of its focus on feature importance. But some of the generic feature importance measurements that @amueller has been talking about might belong with it. Perhaps: `sklearn.interpret` (though we don't want to blow out our scope too much by claiming interpretation is something we do in general), `sklearn.inspect`, `sklearn.inspection`, `sklearn.importance`

Reply:
Our subpackage names are certainly getting more abstract with things like `compose` and `inspect`!

Reply:
I am in favor of a `inspect` module. If `partial_dependence` is a top level module, what other functions/classes would we want to include in it?

Reply:
+1 for `inspect` as well. Shall I do it now?